aws/terraform/metaflow/example.tfvars

```diff
@@ -0,0 +1,21 @@
+env = "prod"
+
+aws_region = "us-west-2"
+
+# Since we're testing, as this is the dev stack setting the min and desired to 0 so that we experience a cold-boot
+# should be appropriate
+cpu_max_compute_vcpus     = 128
+cpu_min_compute_vcpus     = 0
+cpu_desired_compute_vcpus = 0
+
+large_cpu_max_compute_vcpus     = 128
+large_cpu_min_compute_vcpus     = 0
+large_cpu_desired_compute_vcpus = 0
```

Comment:
Rather than have a distinction between `normal`, `large` and `gpu` instances - can we just provide the capability for the user to list instance types that should be part of the compute environment. The user can then invoke multiple instances of the AWS Batch template to generate as many compute environments as they would like.

Reply:
Linking to the comment chain that relates to this so we can have a single discussion thread:

https://github.com/Netflix/metaflow-tools/pull/26#discussion_r631786895